Earthfile

```diff
@@ -614,8 +614,8 @@ arm-image:
     RUN /build-arm-image.sh --use-lvm --model $MODEL --directory "/build/image" /build/$IMAGE_NAME
   END
   IF [ "$COMPRESS_IMG" = "true" ]
-      RUN xz -v /build/$IMAGE_NAME
-      SAVE ARTIFACT /build/$IMAGE_NAME.xz img AS LOCAL build/$IMAGE_NAME.xz
```

Comment:
is this change intended? If yes, maybe we can have a different flag to decide whether we want a qcow2 or img

Reply:
done!